src/commonMain/kotlin/exchange.dydx.abacus/output/Configs.kt

```diff
@@ -247,6 +249,105 @@ data class FeeTier(
     }
 }
 
+@JsExport
+@Serializable
+data class EquityTiers(
+    val shortTermOrderEquityTiers: List<EquityTier>,
```

Comment:
hmm should this more accurately be `IList<EquityTier>`?

Reply:
mmm it seems like they're aliases so IList = List? but i'll change it!